src/eth1spec/number.py

```diff
@@ -196,4 +196,4 @@ def __ipow__(self, right: int, modulo: Optional[int] = None) -> "Uint":
     # TODO: Implement and, or, xor, neg, pos, abs, invert, ...
 
     def to_big_endian(self) -> bytes:
-        raise NotImplementedError()  # TODO
+        return bytes(bytearray.fromhex(hex(self).lstrip("0x")))
```

Comment:
Can you use `self.to_bytes(X, "big")` here?

Reply:
Unfortunately not, if I do it gets grumpy during the `ecrecover`:

```
def to_big_endian(self) -> bytes:
>       return self.to_bytes(self, "big")
E       OverflowError: Python int too large to convert to C ssize_t

src/eth1spec/number.py:199: OverflowError
============================================== short test summary info ==============================================
FAILED tests/test_spec.py::test_add - OverflowError: Python int too large to convert to C ssize_t
```

Any other recommendations?

Reply:
I think you can do something with the `length` argument:

```python
>>> 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF.to_bytes(32, 'big')
b'\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff'
```

Reply:
add shoot, my fault - i didn't mean to have self there